shell/platform/windows/compositor_opengl.h

```diff
@@ -17,7 +17,8 @@ namespace flutter {
 // Enables the Flutter engine to render content on Windows using OpenGL.
 class CompositorOpenGL : public Compositor {
  public:
-  CompositorOpenGL(FlutterWindowsEngine* engine,
+  CompositorOpenGL(PlatformViewManager* manager,
+                   FlutterWindowsEngine* engine,
```

Comment:
Super silly nitpick: I usually order arguments by "importance of types". In this case, I'd consider the engine to be more important and would place it before the platform view manager.